Unreal/SystemLocalized/det/UnrealShare.det

```diff
@@ -707,13 +707,13 @@ MenuName="Vogel"
 ; EN: NameArticle=" a "
 NameArticle=" ein "
 ; EN: MenuName="Biterfish"
-MenuName="Fisch"
+MenuName="Beißfisch"
 
 [BiterfishSchool]
 ; EN: NameArticle=" a "
 NameArticle=" ein "
 ; EN: MenuName="Biterfish"
-MenuName="Fisch"
+MenuName="Beißfisch"
 
 [Brute]
 ; EN: NameArticle=" a "
```

Comment:
@SlyMk3 I like "Schuft".

Reply:
Also "Biest".

Reply:
Or "Untier".

Reply:
Beißfisch??? Why not "Piranha"?

Reply:
@JanUrbansky Piranha wouldn't be in the spirit of biterfish. See also devilfish, which are clearly intended to be special names for them (perhaps for the alien-feel and to avoid Earth-names?).

@ividyon Schuft is not bad, but could sound out of place in message context. Would have to check if there's any translator messages that refer to them. I'm fairly sure there was one near the beginning in NyLeve, but I'm not sure if it dropped a name for them or just described them. Like I mentioned before in one comment: In case of doubt I'd suggest using the original name. Could go with Schuft for now, but I think if it were to be used in messages, Untier is likely the most fitting.